schema/dict.go

```diff
@@ -9,8 +9,13 @@ import (
 type Dict struct {
 	// KeysValidator is the validator to apply on dict keys.
 	KeysValidator FieldValidator
-	// ValuesValidator is the validator to apply on dict values.
-	ValuesValidator FieldValidator
+
+	// ValuesValidator is deprecated, use Values instead to specify a Field. If
+	// used, Values will be set to a required, non-filterable filed on Compile.
+	// ValuesValidator FieldValidator
```

Comment:
We have an _optino_ to add backwards compatibility by keeping the ValuesValidator field until we start on #77, if there is interest for it. It would add some complexity.

Reply:
I.e. I can either comment in or remove this code.